### ISSUE TITLE
After setting up the panel all of my ports closed 

### ISSUE THREAD

**Issue by SnelleJaapie** [pterodactyl-installer/pterodactyl-installer#472]
Maybe this is the whole point of the management panel, but at the moment I am at a bit of a loss here.

I installed pterodactyl on a raspberry pi 5 (yes I am aware this is not ideal, but wanted to give it a shot anyways to see how its like).
At some point I got the panel working and started setting things up (note: I am using a vnc to view my raspberry pi).
While setting up a node and allocating an ip and some ports (25560-25570), after some seconds my connection drops from the VNC viewer, SSH is still working but nothing else.
I had manually setup a minecraft server a couple weeks prior to this, and this also wont connect anymore, saying the adress is already in use.

As this was a home setup I did not use an SSL or domain, just my internal ipv4 address of the Pi.

At the moment I am just looking for help as to how I can reset the blocking of the ports... I have already tried UFW and adding ports manually as it was a deny all setup, but this did not work.

Anyone has any advice as where I should look for? I am not interested in making pterodactyl work, just trying to open my ports again so i can use VNC viewer again, as this is important to me.
Also the ports for e.g. minecraft are also opened in my router, but still no access.

**Comment by RossMdevs** [comment]
Did you use the install script or manually install it?

**If manual**:
Did you enable UFW or IPTABLES?

**Comment by SnelleJaapie** [comment]
@RossMdevs 
I used the automatic setup. I believe during the installation there is an option for automatic firewall rules? I have yet to check what it was exactly.

I also checked UFW and it was indeed active with a couple of port allowed, 22 and the deamon ports 8080 and 2022. 

I tried adding VNC port (5900) and also disabling UFW completely but no luck.

Is the automatic firewall setup different from the manual?
